ddtrace/tracer/span.go

```diff
@@ -337,6 +337,21 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+func (s *span) Format(f fmt.State, c rune) {
+	switch c {
+	case 's':
+		fmt.Fprint(f, s.String())
```

Comment:
Perfect! That way we keep the `String` method for anyone who might've used it, even if it was "hidden away".